src/ledger.test.ts

```diff
@@ -6,7 +6,7 @@ import {
   TEST_FIXTURES,
   ROOT_FINGERPRINT,
   TESTNET,
-  Braid,
+  BraidDetails,
```

Comment:
I don't think `unchained-bitcoin` exports anything called BraidDetails, let alone a type of this name.

Reply:
it does now! https://github.com/unchained-capital/unchained-bitcoin/pull/64/commits/7c952987d113877006be9880531d3e24d64b3bf6#diff-3ffbdbfbe56d0a02febc16d54bb115634a239ba47d8e9f0c49fb0095ad1f1703

Reply:
It will.

Reply:
Ah nvm, the pr we just merged... got it